.github/actions/setup-runner/action.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Setup Runner'
+description: 'Composite action to set up the GitHub Runner for jobs in the test.yml workflow'
+
+runs:
+  using: composite
+  steps:
+    - run: |
+        sudo modprobe ip_vs
+        sudo modprobe ipv6
+        sudo modprobe ip6table_filter
+        sudo modprobe -r overlay
+        sudo modprobe overlay redirect_dir=off
+      shell: bash
+    - run: |
+        if [ ! -e /etc/docker/daemon.json ]; then
+         echo '{}' | tee /etc/docker/daemon.json >/dev/null
+        fi
+        DOCKERD_CONFIG=$(jq '.+{"experimental":true,"live-restore":true,"ipv6":true,"fixed-cidr-v6":"2001:db8:1::/64"}' /etc/docker/daemon.json)
+        sudo tee /etc/docker/daemon.json <<<"$DOCKERD_CONFIG" >/dev/null
+        sudo service docker restart
+      shell: bash
+    - run: |
+        bash ./contrib/check-config.sh || true
```

Comment:
Not sure if it can fail, I just translated what is in Jenkins stage: https://github.com/moby/moby/blob/067062129171dba592638e8e3a0efbfa8a71376b/Jenkinsfile#L81

Reply:
Ah! Good one. I can only assume we did that because we thought it to be "informational". Perhaps, now that we have more separate jobs, we could make it an actual fail (so that we are forced to fix if things go wrong).

Fine for a follow-up though